curator/router.js

```diff
@@ -0,0 +1,5 @@
+const Stations_today = require('./business_logic/stations_today.js')()
+
+module.exports = function (app) {
+  app.get('/stations_today', Stations_today.get_stations_today)
+}
```

Comment:
I have not been really consistent on what variables are capitalized and what are not. Js is not as structured as java when it comes to this. We will need to come up with a consistent solution. I'll put it on trello